js/tooltips.js

```diff
@@ -0,0 +1,121 @@
+import Adapt from 'core/js/adapt';
+import _ from 'underscore';
+import logging from './logging';
+import TooltipView from './views/tooltipView';
+import TooltipModel from './models/TooltipModel';
+
+class TooltipController extends Backbone.Controller {
+
+  initialize() {
+    _.bindAll(this, 'onMouseOver', 'onMouseOut');
+    this._tooltipData = [];
+    this.listenTo(Adapt, 'adapt:preInitialize', this.onAdaptPreInitialize);
+  }
+
+  removeListener() {
+    $(document).off('mouseover', '[data-tooltip-id]', this.onMouseOver);
+  }
+
+  addListener() {
+    this.removeListener();
+    $(document).on('mouseover', '[data-tooltip-id]', this.onMouseOver);
+  }
+
+  getConfig() {
+    return Adapt.course.get('_tooltips');
+  }
+
+  registerGlobalTooltips() {
+    const tooltips = Adapt.course.get('_globals')?._tooltips;
+    if (!tooltips) return;
+    tooltips.forEach(tooltip => this.register(tooltip));
+  }
+
+  getTooltip(id) {
+    return this._tooltipData.find(tooltip => tooltip.get('_id') === id);
+  }
+
+  register(data) {
+    const tooltip = this.getTooltip();
+    if (tooltip) {
+      logging.warn(`Tooltip with id ${tooltip._id} already registered`);
+      return;
+    }
+    this._tooltipData.push(new TooltipModel(data));
+  }
+
+  show(tooltip) {
+    this.hide();
+
+    console.log('show tooltip', tooltip.get('_id'));
+    this.tooltipsView = new TooltipView({model:tooltip, target:this.$mouseoverEl});
+    this.tooltipsView.attach('appendTo', $('body'));
```

Comment:
This was good to live in the dom from the start as you had it before.
The reason is that `aria-live` regions must exists in the DOM and it is their content changes that cause the screen reader to read.

See the 'proceedure' sections of the spec https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA19#procedure

Perhaps hide and show the container instead of adding and removing it?